tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -697,7 +697,7 @@ jobs:
           popd
           wget https://oss.sonatype.org/service/local/repositories/releases/content/org/junit/platform/junit-platform-console-standalone/1.6.2/junit-platform-console-standalone-1.6.2.jar -P ./
           wget https://oss.sonatype.org/service/local/repositories/google-releases/content/com/google/protobuf/protobuf-java/3.9.2/protobuf-java-3.9.2.jar -P ./
-          sudo xcode-select --switch /Applications/Xcode_10.app/Contents/Developer
+          sudo xcode-select --switch /Applications/Xcode_12.4.app/Contents/Developer
```

Comment:
I guess this line was added for "brew install libomp". Now we don't need it .

Reply:
this change is to update the Xcode version to a later one